accounts/forms.py

```diff
@@ -182,7 +182,8 @@ class Meta(LowercaseUsernameUserCreationForm.Meta):
 
 
 class ParticipantSignupForm(LowercaseUsernameUserCreationForm):
-    nickname = forms.CharField(required=True, max_length=255)
+# Rhodri: I don't know why this was here - nickname is in User model
+#    nickname = forms.CharField(required=True, max_length=255)
 
```

Comment:
This doesn't need to be here.  `nickname` is defined in the meta section of the form class.  Please remove it.

Reply:
done